### ISSUE TITLE
Error  MissingGreenlet: greenlet_spawn has not been called when using MySQL

### ISSUE THREAD

**Issue by liquor96** [langflow-ai/langflow#5495]
### Bug Description

If I set the LANGFLOW_DATABASE_URL start with "mysql+aiomysql://"，err MissingGreenlet: greenlet_spawn has not been called; can't call await_only() here. Was IO attempted in an unexpected place?        (Background on this error at: https://sqlalche.me/e/20/xd2s)

### Reproduction

set the environment variable LANGFLOW_DATABASE_URL with "mysql+aiomysql://"。 then execute cmd python -m langflow run

### Expected behavior

langflow works normallly

### Who can help?

No response

### Operating System

window

### Langflow Version

1.1.1

### Python Version

3.11

### Screenshot

![image](https://github.com/user-attachments/assets/3ec11136-b726-409a-a0d9-f0dc7c3727cd)

**Comment by liquor96** [comment]
LANGFLOW_DATABASE_URL is mysql+aiomysql://xxxx:xxxx@x.x.x.x:3306/langflow @dosu

**Comment by MarceloNunesAlves** [comment]
Hi @liquor96, how are you? Have you ever solved this problem?

**Comment by liquor96** [comment]
I tried commenting out this line（await database_service.run_migrations(fix=fix_migration)）in langflow\services\database\util.py line 37, because the migration is synchronous.then run success @MarceloNunesAlves

**Comment by MarceloNunesAlves** [comment]
I believe I have found a solution and have already sent a PR to resolve this.

**Comment by tianzhipeng-git** [comment]
The only thing doesn't work with `aiomysql` is `run_migrations`,  so I create a simple PR to solve this, by one line change. 

The async engine of sqlalchemy need to be executed in greenlet context, so change asyncio.to_thread to greenlet_spawn. They do the same thing - run a sync func in async fashion.

**Comment by tianzhipeng-git** [comment]
> [@tianzhipeng-git](https://github.com/tianzhipeng-git) this should be fixed by [#6258](https://github.com/langflow-ai/langflow/pull/6258) which has been merged. Can you check on main branch ?

Indeed, your solution is better. I tested it locally on MariaDB using the master branch. I have already closed my other PR, and I think this issue can also be closed.

Although there are still some errors, they are related to SQL dialects and are not related to this issue. I gave up on trying to use MySQL as it requires modifying many queries (mainly involving LIMIT/OFFSET)

**Comment by cbornet** [comment]
Thanks for checking.
Yes there are other errors with the MySQL dialect.
MySQL is not officially supported but you can do PRs to make it work if you want.
